# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1662712561 -7200
# Parent  3bce0c18bba296aaad23ced89d65efbe3ba51aae
Misc: Don't allow language pack download from Thunderbird site.

diff --git a/mail/app/profile/all-thunderbird.js b/mail/app/profile/all-thunderbird.js
--- a/mail/app/profile/all-thunderbird.js
+++ b/mail/app/profile/all-thunderbird.js
@@ -865,26 +865,20 @@ pref("intl.regional_prefs.use_os_locales
 // Multi-lingual preferences:
 //  *.enabled - Are langpacks available for the build of Firefox?
 //  *.downloadEnabled - Langpacks are allowed to be downloaded from ATN. ATN only serves
 //      langpacks for release and beta. There is no release-only define, so we also enable
 //      it for beta.
 //  *.liveReload - Switching a langpack will change the language without a restart.
 //  *.liveReloadBidirectional - Allows switching when moving between LTR and RTL
 //      languages without a full restart.
-pref("intl.multilingual.enabled", true);
-#if defined(RELEASE_OR_BETA)
-pref("intl.multilingual.downloadEnabled", true);
+pref("intl.multilingual.enabled", true); // Allow manual or distro installation of langpacks.
+pref("intl.multilingual.downloadEnabled", false); // Don't allow download from Thunderbird site.
 pref("intl.multilingual.liveReload", false);
 pref("intl.multilingual.liveReloadBidirectional", false);
-#else
-pref("intl.multilingual.downloadEnabled", false);
-pref("intl.multilingual.liveReload", false);
-pref("intl.multilingual.liveReloadBidirectional", false);
-#endif
 
 // if true, use full page zoom instead of text zoom
 pref("browser.zoom.full", true);
 
 pref("toolkit.osKeyStore.loglevel", "Warn");
 
 // Developer Tools related preferences
 pref("devtools.chrome.enabled", true);
